ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -120,6 +120,68 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-ipi-byo-iam-role-efs-fips-f60
```

Comment:
@lwan-wanglin we can select one job for fips, another for non-fips from a pair of ci jobs.

Reply:
ok, I will keep non-fips option when moving the arm64 jobs to amd64 ones.